addons/cds-connector-0.0.1/chart/cds-connector/templates/cm-spec.yaml

```diff
@@ -0,0 +1,127 @@
+                "summary": "New order created",
+                "payload": {
+                  "type": "object",
+                  "required": [
+                    "action"
+                  ],
+                  "properties": {
+                    "profileId": {
+                      "type": "string"
+                    },
+                    "orderId": {
+                      "type": "string"
+                    }
+                  }
+                }
+              }
+            },
+            "profiles.affinities": {
+              "subscribe": {
+                "summary": "Affinity is updated",
+                "payload": {
+                  "type": "object",
+                  "required": [
+                    "action"
+                  ],
+                  "properties": {
+                    "profileId": {
+                      "type": "string"
+                    }
+                  }
+                }
+              }
+            },
+            "profiles.transactions": {
+              "subscribe": {
+                "summary": "Transaction is updated",
+                "payload": {
+                  "type": "object",
+                  "required": [
+                    "action"
+                  ],
+                  "properties": {
+                    "profileId": {
+                      "type": "string"
+                    }
+                  }
+                }
+              }
+            },
+            "profiles.identities": {
+              "subscribe": {
+                "summary": "identity is updated",
+                "payload": {
+                  "type": "object",
+                  "required": [
+                    "action"
+                  ],
+                  "properties": {
+                    "profileId": {
+                      "type": "string"
+                    }
+                  }
+                }
+              }
+            },
+            "context.commerce.campaignStarted": {
+              "subscribe": {
+                "summary": "identity is updated",
+                "payload": {
+                  "type": "object",
+                  "required": [
+                    "action"
+                  ],
+                  "properties": {
+                    "profileId": {
+                      "type": "string"
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      },
+      "documentation": {
+        "displayName": "SAP CDS Events Documentation",
+        "description": "SAP CDS Events Documentation",
+        "type": "API Doc",
+        "docs": [
+          {
+            "title": "SAP CDS events",
+            "type": "Link",
+            "source": "https://www.sap.com/products/commerce-cloud-context-driven-services.html"
+          }
+        ]
+      }
+    }
+  headers.json: |
+    {
+        "Authorization" : {{ printf "token %s" .Values.token | quote }}
+    }
+kind: ConfigMap
```

Comment:
please move the metadata block to the beginning of the file, that will improve readability.